docs/rule/no-curly-component-invocation.md

```diff
@@ -72,10 +72,13 @@ look like they could be component invocations.
 
 ### Configuration
 
+- boolean -- if `true`, default configuration is applied
+  (`noImplicitThis: false`, `requireDash`: true), see below for details
```

Comment:
The backticks are slightly inconsistent around each parameter.

Reply:
Thank you!